archon-ui-main/docker-compose.test.yml

```
@@ -0,0 +1,53 @@
+      dockerfile: Dockerfile.test
+    container_name: archon-ui-lint
+    volumes:
+      - ./src:/app/src:ro
+    command: npm run lint
+    profiles:
+      - lint
+
+# Named volumes for persisting test results between runs
+volumes:
+  test-results:
+  node_modules:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)